### PR TITLE
feat: add eject command

### PR DESCRIPTION
**Pull request by yannbf**
Issue: #42 

Running eject for the first time:
image

And in case the config file already exists, it will notify the user (but still override it without prompting)
image

### Improvement points/challenges for later

- Using the test-runner in monorepos. Very likely the logic to detect and use the config file is not ready for that
- Adding inquirer or any other library to allow for prompting user to things like (are you sure you want to override ; we detected an external storybook url, do you want to use stories-json flag)

<details>
  <summary>📦 Published PR as canary version: <code>0.0.3-canary.54.04b73e7.0</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @storybook/test-runner@0.0.3-canary.54.04b73e7.0
  # or 
  yarn add @storybook/test-runner@0.0.3-canary.54.04b73e7.0
  ```
</details>

### CODE REVIEW

File: bin/test-storybook.js [hunk 1]
```diff
@@ -108,12 +108,30 @@ async function fetchStoriesJson(url) {
   return tmpDir;
 }
 
+function ejectConfiguration () {
+  const origin = path.resolve(__dirname, '../playwright/test-runner-jest.config.js')
+  const destination = path.resolve('test-runner-jest.config.js')
+  const fileAlreadyExists = fs.existsSync(destination)
+  
+  if(fileAlreadyExists) {
+    console.log('[test-runner] Found existing file, overriding...')
```
Comment (shilman) [inline bin/test-storybook.js:117]: Should we error here instead?

**Review by shilman — APPROVED** [review]
LGTM! 🚀

File: bin/test-storybook.js (hunk 1 above)
Comment (yannbf) [reply bin/test-storybook.js:117]: I thought this could be further improved with a prompt for yes/no to override. Do we want to make it fail?

Comment (shilman) [reply bin/test-storybook.js:117]: I think we should just tell them to delete the file & re-run

Comment (shilman) [reply bin/test-storybook.js:117]: Prompt if you're feeling extra fancy 😄

Comment (yannbf) [reply bin/test-storybook.js]: Sounds good. Done!

File: bin/test-storybook.js [hunk 2]
```diff
@@ -114,7 +114,7 @@ function ejectConfiguration () {
   const fileAlreadyExists = fs.existsSync(destination)
   
   if(fileAlreadyExists) {
-    console.log('[test-runner] Found existing file, overriding...')
+    throw new Error('Found existing file. Please delete it and rerun this command')
```
Comment (shilman) [inline bin/test-storybook.js:117]: print file location? @yannbf

**Pull request merged**